explorer/app/entity/api/service.ts

```diff
@@ -18,38 +19,58 @@ import { convertUrlParams } from "../../utils/url";
  * Request to get a list of entities.
  * @param apiPath - Path that will be used to compose the API url
  * @param listParams - Params to be used on the request. If none passed, it will default to page's size 25 and the current catalog version
+ * @param options - String with the type of API call, must be either GET or POST for now
  * @returns @see ListResponseType
  */
 export const list = async (
   apiPath: string,
-  listParams?: AzulListParams
+  listParams?: AzulListParams,
+  options?: Options
 ): Promise<AzulEntitiesResponse> => {
   const params = { ...DEFAULT_LIST_PARAMS, ...listParams };
-  return await fetchList(`${URL}${apiPath}?${convertUrlParams(params)}`);
+  return await fetchList(`${URL}${apiPath}`, params, options);
 };
 
 /**
  * Make a get request to get a list of entities.
```

Comment:
We should probably update the description of this method to something like "Make a get or post..."